uint8 drone_id
geometry_msgs/PoseStamped  point
as2_msgs/TrajectorySetpoints trajectory
bool circular
float32 radius
uint8    control_mode
float32  navigation_speed